be/src/formats/parquet/file_reader.cpp

```diff
@@ -193,9 +195,12 @@ int FileReader::_get_partition_column_idx(const std::string& col_name) const {
 Status FileReader::_decode_min_max_column(const ParquetField& field, const std::string& timezone,
                                           const TypeDescriptor& type, const tparquet::ColumnMetaData& column_meta,
                                           const tparquet::ColumnOrder* column_order, vectorized::ColumnPtr* min_column,
-                                          vectorized::ColumnPtr* max_column) {
+                                          vectorized::ColumnPtr* max_column, bool* decode_ok) {
+    *decode_ok = true;
+
     if (!_can_use_min_max_stats(column_meta, column_order)) {
-        return Status::NotSupported("min max statistics not supported");
```

Comment:
`Status::NotSupported` can be taken as success of `decode_min_max_column` or not. Because right down there, following code could also return `Staus::NotSupported`, but this error is fatal.

```
ColumnConverterFactory::create_converter(field, type, timezone, &converter));
```